src/engraving/libmscore/beam.cpp

```diff
@@ -278,17 +263,17 @@ bool Beam::isNoSlope() const
 
 void Beam::alignBeamPosition()
 {
-    PointF currentBeamPos = beamPos();
+    BeamPos currentBeamPos = beamPos();
```

Comment:
the use of PointF here was beyond comprehension - using `QPair<qreal, qreal>` isn't perfect but it needs to be a type that works with QVariant.

Reply:
What is beyond comprehension there?

Reply:
Because it's two *y* values not x & y.

Reply:
Ah! Now the penny dropped.